### PR TITLE
Changes to active.

### LINKED ISSUE #69: Can't deactivate
Steps to reproduce:
1. Create a collection that is active.
2. Edit that collection and deselect active.
3. Click Update Collection.

Expected result:
Collection is updated with active being false.

Actual result:

### PR DESCRIPTION
**Pull request by justinlittman**
Active is true by default. Validates that active is true or false. Renamed active on collection list page.

closes #69

### CODE REVIEW

File: app/views/collections/index.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <tr>
     <th scope="col">Collection</th>
     <th scope="col">Druid</th>
-    <th scope="col">On/off</th>
+    <th scope="col">Active / inactive</th>
```
Comment (ndushay) [inline app/views/collections/index.html.erb]: should both be capitalized?  Do we like the spaces around the slash?

Comment (justinlittman) [reply app/views/collections/index.html.erb]: Fixed @ndushay

**Comment by ndushay** [comment]
now you have merge conflicts, but feel free to merge after conflicts resolved.

**Comment by mjgiarlo** [comment]
I resolved the merge conflict. Builds are running. :watch:

**Pull request merged**